config/htcondor-ce.spec

```diff
@@ -418,16 +445,21 @@ fi
 %{_datadir}/condor-ce/config.d/02-ce-bosco-defaults.conf
 
 %files client
+%config(noreplace) %{_sysconfdir}/condor-ce/condor_mapfile.osg
+%config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-osg.conf
```

Comment:
The mapfile needs to be in the client because the client wants to authenticate the remote CE as well.